spec/stat_tracker_spec.rb

```diff
@@ -7,19 +7,21 @@
     team_path = './data/teams.csv'
     game_teams_path = './data/game_teams.csv'
 
-    locations = {
+    @locations = {
       games: game_path,
       teams: team_path,
       game_teams: game_teams_path
     }
 
-    @stat_tracker = StatTracker.from_csv(locations)
+    @stat_tracker = StatTracker.from_csv(@locations)
   end
 
   describe '#initialize' do
     it 'exists' do
```

Comment:
I had to add this line in order to hit 100% coverage with simplecov, as before we were only getting 65%.